packages/next/src/server/app-render/app-render.tsx

```diff
@@ -3089,7 +3163,8 @@ async function prerenderToStream(
         implicitTags,
         renderSignal: finalClientController.signal,
         controller: finalClientController,
-        // For HTML Generation we don't need to track cache reads (RSC only)
+        // For HTML Generation the only cache tracked activity
+        // is module loading, which has it's own cache signal
```

Comment:
Both this comment as well as the one it replaces are kinda wrong, aren't they? Should we use the same comment we use for the final server prerender here as well?
> All caches we could read must already be filled so no tracking is necessary

Reply:
The fundamental flaw is the client version of this store just doesn't need a cacheSignal in it's type. I'll address after forking the type. I went with

> No APIs require a cacheSignal through the workUnitStore during the HTML prerender